docs/apm/agent-configuration.asciidoc

```diff
@@ -2,18 +2,15 @@
 [[agent-configuration]]
 === APM Agent configuration
 
-beta[] APM Agent configuration allows you to fine-tune your agent configuration directly in Kibana.
+APM Agent configuration allows you to fine-tune your agent configuration directly in Kibana.
```

Comment:
@bmorelli25 is there any note about kibana 7.5 not being compatible with apm-server < 7.5 (or viceversa) for this feature? or is that a to-do for release notes?

Reply:
@jalvz - I've noted this in three places: 

* [APM Release notes](https://www.elastic.co/guide/en/apm/get-started/7.5/release-highlights-7.5.0.html)
* [Installation and upgrade guide](https://www.elastic.co/guide/en/elastic-stack/7.5/apm-highlights.html)
* [APM Server Breaking changes](https://www.elastic.co/guide/en/apm/server/7.5/breaking-changes.html)

I think that's sufficient, but if you still have concerns I can take another look at these docs.

Reply:
BTW, I added it in initially because of your reminder in the release notes PR, so thank you!

Reply:
more than sufficient, thank you!